### PR TITLE
CB-10764: Remove emoji in cordova-plugin-splashscreen

### PR DESCRIPTION
**Pull request by sarangan12**
Removing warning emoji

@dblotsky @riknoll @rakatyal Can you please review and merge this PR?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@
 
 This plugin displays and hides a splash screen during application launch.
 
-:warning: Report issues on the [Apache Cordova issue tracker](https://issues.apache.org/jira/issues/?jql=project%20%3D%20CB%20AND%20status%20in%20%28Open%2C%20%22In%20Progress%22%2C%20Reopened%29%20AND%20resolution%20%3D%20Unresolved%20AND%20component%20%3D%20%22Plugin%20Splashscreen%22%20ORDER%20BY%20priority%20DESC%2C%20summary%20ASC%2C%20updatedDate%20DESC)
+Report issues with this plugin on the [Apache Cordova issue tracker][Apache Cordova issue tracker]
```
Comment (dblotsky) [inline README.md]: Please add a period at the end.

**Comment by riknoll** [comment]
The commit message has a typo

File: README.md (hunk 1 above)
Comment (sarangan12) [inline README.md]: Done

**Comment by rakatyal** [comment]
LGTM.

**Pull request merged**